content/index.rst

```diff
@@ -68,6 +68,33 @@ Software for Humans
 
     Let's talk about your project
 
+.. callmacro:: content.macros.j2#rst
+  :tag: 'start'
+
+.. callmacro:: home.macros.j2#expertise
+
+  While we’ve become industry leaders –
+  experts and core developers of languages like `Sass/CSS`_ and
```

Comment:
If we're linking the language names individually, I feel like they should  have different targets. We could either add in-page ID links, or…?

Reply:
You know, I wasn't sure where I should point these to so I just copied what we did further down the page. I also was hoping for an anchor link at minimum but didn't see any available and since I wasn't sure if that was even where we wanted to go with these link I didn't take time to add an ID

Reply:
@SondraE do we need links here? Are they useful? Or should we add detail pages for the main OSS projects we highlight?

Reply:
@stacyk @mirisuzanne 
Hmm. I don't think we really need links there. However, I do like how the links highlight those areas of expertise. It's like a quick summary of our language specialities before people scroll all the way down to the "performant python..." section. Is there another good way to highlight those? Bold weight is already being used in that section and more bold text won't stand out much. 

It would be nice to have detail pages for our main OSS projects, but that seems like a lower priority than some other OddSite updates in the Trello list.

Reply:
Quick solution, we could maybe merge them into a single `Sass/CSS and Python/Django` link? I'll merge, and we can hotfix if that sounds good to you @SondraE @stacyk?